doc/manual/src/glossary.md

```diff
@@ -217,6 +217,19 @@
 
   [output path]: #gloss-output-path
 
+- [deriving path]{#gloss-deriving-path}
+
+  A [store]-level expression (not Nix language expression) that denotes a [store object].
+  There are two forms:
+
+  - *constant*: A constant deriving path is just a [store path]
```

Comment:
```suggestion
  - *constant*: just a [store path]
```

These are many words that add no information.

It's not wrong to have "constant deriving path" in a sentence somewhere, but we'll likely find more opportunities for that.